src/processing/dom/html-namespaces.js

```diff
@@ -0,0 +1,13 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+
+export default {
+    HTML:   'http://www.w3.org/1999/xhtml',
+    MATHML: 'http://www.w3.org/1998/Math/MathML',
+    SVG:    'http://www.w3.org/2000/svg',
+    XLINK:  'http://www.w3.org/1999/xlink',
+    XML:    'http://www.w3.org/XML/1998/namespace',
+    XMLNS:  'http://www.w3.org/2000/xmlns/'
+};
```

Comment:
It named as in https://www.w3.org/TR/2011/WD-html5-20110405/namespaces.html.
So, in parse5 it named simular (https://github.com/inikulin/parse5/blob/24cb16c4a46f84fba01c6f41fc96817a013a6495/lib/common/html.js).